.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.1.14
+  hooks:
+    # Run the linter.
+  - id: ruff
+    # Run the formatter.
+  - id: ruff-format
```

Comment:
Run the formatter in `--check` mode.

Reply:
Allows loading of unsafe yaml tags, which might execute python code. Not needed here -> removed